drainer/config.go

```diff
@@ -65,6 +65,9 @@ type SyncerConfig struct {
 	IgnoreSchemas     string             `toml:"ignore-schemas" json:"ignore-schemas"`
 	IgnoreTables      []filter.TableName `toml:"ignore-table" json:"ignore-table"`
 	TxnBatch          int                `toml:"txn-batch" json:"txn-batch"`
+	MarkStatus        bool               `toml:"mark-status" json:"mark-status"`
+	DdlSync           bool               `toml:"ddl-sync" json:"ddl-sync"`
```

Comment:
how about use https://github.com/pingcap/tidb-tools/blob/06b27bff7c2e29aac542b2e7197a52ae96221142/pkg/binlog-filter/filter.go#L85 to support more function? it is not necessary, we can do this later